### DISCUSSION TITLE
Incoming Message tidak muncul di Webhook setelah dideploy ke server

### DISCUSSION THREAD

**Discussion by engk0ng** [kirimwa-id/forum#171] [Tanya Jawab] [answered]
Bismillah..webhook saya bisa menerima incoming message saat di localhost (pake url Ngrok), tapi setelah dideploy ke server jadi tidak muncul incoming messagenya...padalah jika di curl (req post) ke alamat webhooknuya data bisa masuk...kira-kira kenapa ya? Terima kasih.

**Comment by rioastamal** [comment, marked as answer]
Cek request dan response-nya di https://developer.kirimwa.id/api/v1/#get-webhook-responses

**Reply by engk0ng** [reply]
Terima kasih mas rio atas responnya. Sudah saya cek, ternyata http_responsenya Error: unable to verify the first certificate...

**Comment by engk0ng** [comment]
Sudah clear. Terim kasih

**Comment by masbagusstwn** [comment]
Kalau sudah punya server apakah alangkah lebih baiknya deploy whatsapp module seperti bailey dan model2nya gitu?

**Comment by sorryjack** [comment]
saya mengalami masalah, webhook tersimpan dan aktive statusnya. tapi tidak menerima apapun. Saya coba test juga pakai webhook.site dan hasilnya sama. tidak ada request masuk.

Mohon bantuannya gan...

**Reply by rioastamal** [reply]
@sorryjack, cek dulu https://developer.kirimwa.id/api/v1/#get-webhook-responses

**Reply by sorryjack** [reply]
sudah.. selalu kosong seperti ini response nya
![image](https://github.com/kirimwa-id/forum/assets/6811578/8eb66dfc-c093-4b42-8b7c-09404917e3c9)
